fuzzing/common.bzl

```diff
@@ -15,19 +15,23 @@
 
 """This file contains common rules for fuzzing test."""
 
+load("@bazel_skylib//rules:common_settings.bzl", "BuildSettingInfo")
+
 def _fuzzing_launcher_impl(ctx):
     # Generate a script to launcher the fuzzing test.
     script = ctx.actions.declare_file("%s" % ctx.label.name)
 
     script_template = """#!/bin/sh
-exec {launcher_path} {target_binary_path} --corpus_dir={corpus_dir} "$@"
-"""
+exec {launcher_path} {target_binary_path} --corpus_dir={corpus_dir} --engine={engine_type} "$@" """
 
     script_content = script_template.format(
         launcher_path = ctx.executable._launcher.short_path,
         target_binary_path = ctx.executable.target.short_path,
         corpus_dir = ctx.file.corpus.short_path if ctx.attr.corpus else "",
+        engine_type = ctx.attr._engine[BuildSettingInfo].value,
     )
+    if ctx.attr.is_regression:
+        script_content += " --regression=True"
```

Comment:
This approach can trick the reader to believe the `script_template` above is actually the entire script, whereas we may be adding extra content to it. Moreover, assuming that `script_content` ends in a way we can just append an argument to it also seems a bit brittle.

Consider constructing the command line for the script in a consistent manner. For example, you can first construct a vector of arguments for the launcher command line, and flatten that to a string, then substitute that string into the script template.